.travis.yml

```diff
@@ -35,3 +35,10 @@ script:
     | grep -q 'The PHP Group'
     && (echo 'PHP is installed' && exit 0)
     || (echo 'PHP is not installed' && exit 1)
+
+  # Make sure PHP cli is correctly configured.
+  - >
+    php -r "echo ini_get('date.timezone'), \"\\n\";"
+    | grep -q 'Europe/London'
```

Comment:
It must be different from 'America/Chicago' for differentiate the CLI environment from the web environment.